lib/hackerone/client/report.rb

```diff
@@ -75,6 +75,32 @@ def activities
         end
       end
 
+      def program
+        @program || Program.find(relationships[:program][:data][:attributes][:handle])
+      end
+
+      def assign_to_user(name)
+        fail(
```

Comment:
Makes sense and works for me! We (as @hacker0x01) should fix the API and return useful error responses.